core/kazoo_stdlib/src/kz_module.erl

```diff
@@ -17,12 +17,15 @@ is_exported(Module, Function, Arity)
        is_atom(Function),
        is_integer(Arity),
        Arity >= 0 ->
-    case erlang:module_loaded(Module) of
-        'false' -> _ = code:ensure_loaded(Module), 'ok';
-        'true' -> 'ok'
-    end,
-    erlang:function_exported(Module, Function, Arity).
-
+    case ensure_loaded(Module) of
+        'false' -> 'false';
+        Module -> erlang:function_exported(Module, Function, Arity)
+    end;
+is_exported(Module, Function, Arity) ->
+    is_exported(kz_term:to_atom(Module)
```

Comment:
should we call kz_term:to_atom(Module, 'true') ?

Reply:
@lazedo I didn't want to since this is used in crossbar to check path tokens, so every token on the path will be prefixed with cb_ and tested here. I'd rather the `to_atom/1` fail and have the caller handle that

Reply:
got it, and i makes sense (saw that in the mkbusy tests). maybe catch the exception here ? that would make the call to `kz_module:is_module` safe.